cmd/docker-app/install.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/deislabs/duffle/pkg/action"
+	"github.com/deislabs/duffle/pkg/bundle"
+	"github.com/deislabs/duffle/pkg/claim"
+	"github.com/deislabs/duffle/pkg/credentials"
+	"github.com/deislabs/duffle/pkg/utils/crud"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type installOptions struct {
+	parametersOptions
+	credentialOptions
+	orchestrator     string
+	namespace        string
+	kubeNamespace    string
+	stackName        string
+	insecure         bool
+	sendRegistryAuth bool
+}
+
+type nameKind uint
+
+const (
+	_ nameKind = iota
+	nameKindEmpty
+	nameKindFile
+	nameKindDir
+	nameKindReference
+)
+
+const longDescription = `Install the application on either Swarm or Kubernetes.
+Bundle name is optional, and can:
+- be an empty and resolves to any *.dockerapp in working directory
+- be an BUNDLE file path and resolves to any *.dockerapp file or dir, or any CNAB file (signed or unsigned)
```

Comment:
`s/an/a/`

Also, why `BUNDLE` in capitals?

Reply:
resolves→resolve here too

Reply:
👍